sonar-update-center-common/src/main/java/org/sonar/updatecenter/common/UpdateCenterDeserializer.java

```diff
@@ -235,22 +235,33 @@ private void parsePluginReleases(Properties p, Sonar sonar, String pluginKey, Pl
     }
   }
 
-  private Release parsePluginRelease(Properties p, Sonar sonar, String pluginKey, Plugin plugin, boolean isPublicRelease, boolean isArchivedRelease, String pluginVersion) {
+  private Release parsePluginRelease(Properties p
+      , Sonar sonar
```

Comment:
Not sure my previous comment was preserved so I'm repeating here: having comma upfront looks strange. Our coding conventions are probably not precise enough but that the first time I see that in our codebase.